### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by NellaMori**
#### Description
Changes proposed in this pull request:

* Add cards on Homepage
* Add information about dummy users

#### What I have learned working on this feature: 
* Show objects in arrays with *NgFor

#### Screenshots:
schermata 2018-04-05 alle 15 30 57

### CODE REVIEW

File: src/pages/home/home.ts [hunk 1]
```diff
@@ -7,8 +7,53 @@ import { NavController } from 'ionic-angular';
 })
 export class HomePage {
 
+  users: any
+  user: any
   constructor(public navCtrl: NavController) {
```
Comment (AKidd95) [inline src/pages/home/home.ts]: Looks like you are only using the ```users: any``` variable and not the ```user: any```

**Review by AKidd95 — CHANGES_REQUESTED** [review]
looks good besides the one unnecessary variable

File: src/pages/home/home.ts (hunk 1 above)
Comment (NellaMori) [reply src/pages/home/home.ts]: good catch, deleted.

**Review by isabellelii — APPROVED** [review]

**Review by AKidd95 — APPROVED** [review]
Good job guys

**Pull request merged**